packages/module/src/VirtualAssistant/VirtualAssistant.tsx

```diff
@@ -115,7 +140,7 @@ export interface VirtualAssistantProps {
 export const VirtualAssistant: React.FunctionComponent<VirtualAssistantProps> = ({
   children,
   title = 'Virtual Assistant',
-  inputPlaceholder = 'Type a message...',
+  inputPlaceholder = 'Send a message...',
```

Comment:
cypress test code is looking for Type a message: https://github.com/patternfly/virtual-assistant/blob/main/cypress/component/VirtualAssistant.cy.tsx#L9C97-L9C111

Could you update it so it looks for `Send a message...` instead?